examples/glusterfs/README.md

```diff
@@ -0,0 +1,42 @@
+## Glusterfs
+
+[Glusterfs](http://www.gluster.org) is an open source scale-out filesystem. These examples provide information about how to allow Docker containers use Glusterfs volumes.
```

Comment:
nit: I assume the current implementation is not docker dependent. It should work in any kubernetes cluster irrespective of the underlying containment technology.